src/SymfonyCache/CleanupCacheTagsListener.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache\SymfonyCache;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Listener that allows to cleanup the cache tags header so it's not exposed
+ * to the world.
+ *
+ * @author Yanick Witschi <yanick.witschi@terminal42.ch>
+ */
+class CleanupCacheTagsListener implements EventSubscriberInterface
+{
+    /**
+     * @var string
+     */
+    private $tagsHeader;
+
+    /**
+     * @param string $tagsHeader The header that is used for cache tags
+     */
+    public function __construct($tagsHeader = PurgeTagsListener::DEFAULT_TAGS_HEADER)
```

Comment:
its the same value, but this should be `TagHeaderFormatter::DEFAULT_HEADER_NAME` because its about the header used to mark tags on a response. the purge constant is about which header is used to tell what tags to purge... its a bit confusing that we chose the same default for both those headers...